Source/Core/DolphinQt2/ToolBar.cpp

```diff
@@ -93,13 +94,22 @@ void ToolBar::UpdateIcons()
 {
   QString dir = Settings().GetThemeDir();
 
-  m_open_action->setIcon(QIcon(QStringLiteral("open.png").prepend(dir)));
-  m_paths_action->setIcon(QIcon(QStringLiteral("browse.png").prepend(dir)));
-  m_play_action->setIcon(QIcon(QStringLiteral("play.png").prepend(dir)));
-  m_pause_action->setIcon(QIcon(QStringLiteral("pause.png").prepend(dir)));
-  m_stop_action->setIcon(QIcon(QStringLiteral("stop.png").prepend(dir)));
-  m_fullscreen_action->setIcon(QIcon(QStringLiteral("fullscreen.png").prepend(dir)));
-  m_screenshot_action->setIcon(QIcon(QStringLiteral("screenshot.png").prepend(dir)));
-  m_config_action->setIcon(QIcon(QStringLiteral("config.png").prepend(dir)));
-  m_controllers_action->setIcon(QIcon(QStringLiteral("classic.png").prepend(dir)));
+  m_open_action->setIcon(
```

Comment:
Same as above with the common logic. Maybe this logic could be moved into the Resources namespace as well?